### PR TITLE
Improve stdout/stderr silencing from #260 and #253

### PR DESCRIPTION
**Pull request by ddworken**
#260 #253

### CODE REVIEW

**Comment by ccoVeille** [comment]
For records, issues mentioned in PR title are not linked on GitHub. So right now, nothing appears on these PR

**Comment by ddworken** [comment]
> For records, issues mentioned in PR title are not linked on GitHub. So right now, nothing appears on these PR

Ah, thanks! I added them to the description to fix the linking.

File: client/lib/config.zsh [hunk 1]
```diff
@@ -29,7 +29,7 @@ function _hishtory_precmd() {
     fi
     (hishtory saveHistoryEntry zsh $_hishtory_exit_code "$_hishtory_command" $_hishtory_start_time &) 2>&1 >/dev/null  # Background Run
     # hishtory saveHistoryEntry zsh $_hishtory_exit_code "$_hishtory_command" $_hishtory_start_time 2>&1 >/dev/null # Foreground Run
-    (hishtory updateLocalDbFromRemote &)
+    (hishtory updateLocalDbFromRemote &) 2>&1 >/dev/null
```
[the remark this answers is not in the document]
Comment (ddworken) [reply client/lib/config.zsh:32]: Both seem to work fine for me:

```
david@Davids-Air hishtory % (echo hi &)
hi
david@Davids-Air hishtory % (echo hi > /dev/null &)
david@Davids-Air hishtory % (echo hi &) > /dev/null
```

So unless there is a reason to use one over the other, I'm inclined to stick with the way I wrote it which I see as slightly clearer? Though I'm open to your feedback/thoughts. :)

[the remark this answers is not in the document]
Comment (ccoVeille) [reply client/lib/config.zsh:32]: Thanks for checking.

My main concern is now the fact you should use the following syntax then

```suggestion
    hishtory updateLocalDbFromRemote 2>&1 >/dev/null &
```

A syntax commonly understood

[the remark this answers is not in the document]
Comment (ddworken) [reply client/lib/config.zsh:32]: Hmm, so I do have a vague memory of purposefully using a subshell here, though I can't seem to remember why. I probably should have documented this when I added the subshell here. I'm going to leave this as-is for now, but I'll open a separate CL to drop the sub-shell and see if that passes the GH action tests. If it does, I'll merge it separately. So thank you for the feedback on this! And please do feel free to send any more feedback/comments/PRs my way. :)

**Pull request merged**